testing/i3blocks/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+pkgname=i3blocks
+pkgver=20190321_git
+pkgrel=0
+pkgdesc="A minimalist scheduler for your status line scripts"
+url="https://github.com/vivien/i3blocks"
+arch="all"
+license="GPL"
+makedepends="autoconf automake"
+subpackages="$pkgname-doc"
+_commit="ec050e79ad8489a6f8deb37d4c20ab10729c25c3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/vivien/i3blocks/archive/$_commit.tar.gz"
```

Comment:
```suggestion
source="https://github.com/vivien/i3blocks/releases/download/$pkgver/i3blocks-$pkgver.tar.gz
        fix-musl.patch
        "
```

Reply:
please use this [upstream patch](https://github.com/vivien/i3blocks/commit/28c2b092f7d41aa723dad18b0484dca0da1cbbf5.patch) and rename to `fix-musl.patch`  instead of using the commit hash